ui/ui-expenses/src/screens/list/list.reactions.js

```diff
@@ -1,9 +1,16 @@
-import pica from 'pica/dist/pica'
-
 export const load = (action, store, { http }) => {
   http('EXPENSES').get('/api/expenses')
 }
 
+export const postImage = async ({ payload }, store, { http, pica }) => {
+  // compress and resize by 2 the image
+  const imageReduced = await pica.reduce(payload, 2)
+  http('IMAGES').post('/api/images', {
+    image: imageReduced,
```

Comment:
I think, it's more logic to get an image reduced by driver instead of : 
 - catch `ON_SUBMIT`
 - drivers dispatch an event 
 - catch them on other reaction to post the image.
 -